src/utils/api.js

```diff
@@ -0,0 +1,10 @@
+export const fetchTicket = async ticketID => {
+  try {
+    const res = await fetch(`/api/tickets/${ticketID}`, {
+      method: 'GET',
+    });
+    return await res.json();
```

Comment:
don't need to `await` a synchronous call. consider promise chaining.

Reply:
the res.json() is asyncronous, are you talking about that or something else?

screenshot from the [fetch documentation](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch)
![Screen Shot 2019-11-12 at 7 50 47 PM](https://user-images.githubusercontent.com/27692412/68725978-babd9f80-0585-11ea-839d-1b2b060cdaee.png)